pygenometracks/tracks/BedTrack.py

```diff
@@ -118,6 +126,19 @@ def __init__(self, *args, **kwarg):
             else:
                 self.colormap = self.properties['color']
 
+        # check the display is valid:
+        if self.properties['display'] in ['interlaced', 'domain']:
```

Comment:
can you define both valid/invalid ontop of the file after the imports with capital letters ... something like ... DISPLAY_BED_VALID = [...] and DISPLAY_BED_INVALID = [..] then we can reuse them and have a central place to change it....

Reply:
Done